tests/src/Unit/DataService/Media/MediaTest.php

```diff
@@ -63,6 +64,27 @@ public function testInstanceOf($field, $class)
         }
     }
 
+    /**
+     * Test the getNormalizedDefaultThumbnailUrl method.
+     *
+     * The getNormalizedDefaultThumbnailUrl method is unlike the others in that
+     * it does not have a direct property mapping in the MPX data.
+     */
+    public function testGetNormalizedDefaultThumbnailUrl()
+    {
+        // Test against the default media fixture loaded in the setup method.
+        $object = $this->deserialize($this->class, 'defaultThumbnailUrl');
+        $this->assertEquals(new Uri('ftp://winnas01.theplatform.com/Content/Video/thePlatform_Documentation_-_SA/3/494/ian_1_flexible_320x240.jpg'), $object->getNormalizedDefaultThumbnailUrl());
```

Comment:
😱 ftp!

Reply:
I know right?! I wondered where you got that test object from when I saw this :laughing: